.github/workflows/ci.yaml

```diff
@@ -27,6 +27,11 @@ jobs:
             os: ubuntu-latest
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Install Protoc
+        uses: arduino/setup-protoc@ab6203da1c3118e4406048171b09238ad31ad73e # latest
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
```

Comment:
Sorry for the external required dependency. Going away soon once https://github.com/libp2p/rust-libp2p/pull/3312 is merged and released.

Reply:
Thank you for the heads up and a link to the PR; I'll keep track of progress there. 

Depending on `protoc` is not a big deal, but I am happy to remove it as soon as it's no longer needed.